packages/synapse-react-client/src/components/ChallengeDataDownload/ChallengeDataTable.tsx

```diff
@@ -284,9 +284,7 @@ export const ChallengeDataTable: React.FunctionComponent<DetailsViewProps> = ({
           }}
         >
           <Checkbox
-            label="Select All"
-            hideLabel={true}
-            className="SRC-pointer-events-none"
```

Comment:
For some reason, the bootstrap-4-backport wrapper class gave these checkboxes the `cursor: text` style, so I think that's why they had `pointer-events: none` (which was a strange decision that was probably made by me).

I removed the bootstrap-4-backport class and now the cursor works properly.